### PR TITLE
[WIP] add notification files for OpenJDK

### PR DESCRIPTION
**Pull request by riuvshin**
### What does this PR do?
set fixed OpenJDK version (**8u131-b11**) and add notification files to images with OpenJDK 

Following list illustrate dependencies between images and state of adding notification files.

Added notification file:
- alpine_jdk8      
- stack-base:debian
  - node (FROM eclipse/stack-base:debian)
  - debian_jdk8 (FROM eclipse/stack-base:debian)
    - debian_jdk8_node (FROM eclipse/debian_jdk8)
- stack-base:ubuntu
  - selenium (FROM eclipse/stack-base:ubuntu)
  - php (FROM eclipse/stack-base:ubuntu)
  - ubuntu_jdk8 (FROM eclipse/stack-base:ubuntu)
    - ubuntu_gradle (FROM eclipse/ubuntu_jdk8)
    - hadoop-dev (FROM eclipse/ubuntu_jdk8)
    - x11_vnc (FROM eclipse/ubuntu_jdk8)
  - ubuntu_android (FROM eclipse/stack-base:ubuntu)
  - ubuntu_python (FROM eclipse/stack-base:ubuntu)
    - platformio (FROM eclipse/ubuntu_python:2.7)
  - ruby_rails (FROM eclipse/stack-base:ubuntu)
  - ubuntu_go (FROM eclipse/stack-base:ubuntu)
  - ubuntu_wildfly8 (FROM eclipse/stack-base:ubuntu)
  - meteor (FROM eclipse/stack-base:ubuntu)
  - cpp_gcc (FROM eclipse/stack-base:ubuntu)

- dotnet_core (FROM debian:jessie)

No JDK installed / needed:
- mysql

It seems we are fine with CentOS / Fedora stacks as we install OpenJDK from CentOS / Fedora certified repositories. **(must be confirmed)**
CentOS:
- centos_go
- centos
- centos_jdk8
- centos_wildfly_swarm
- centos_vertx
- centos_spring_boot	
- centos_nodejs

Fedora:
- artik

### What issues does this PR fix or reference?
https://github.com/codenvy/planning/issues/129

### CODE REVIEW

File: recipes/selenium/Dockerfile [hunk 1]
```diff
@@ -30,7 +30,6 @@ RUN sudo apt-get update -qqy && \
   x11vnc \
   xvfb \
   subversion \
-  openjdk-8-jdk-headless \
```
Comment (benoitf) [inline recipes/selenium/Dockerfile]: removing jdk for selenium is ok ?

Comment (riuvshin) [inline recipes/selenium/Dockerfile]: that image is `FROM eclipse/stack-base:ubuntu` so it already have jdk from parent it was just redundant

**Comment by TylerJewell** [comment]
The approach is ok.  But we should wait for Patrick's comments on the official way to install packages from Red Hat.  For any package that we can install from an official repository, we would not need to include the custom notification file, as those packages already include it within the official installation.

**Comment by riuvshin** [comment]
@TylerJewell Please note that we will not be able to install RedHat packages on ubuntu / debian / alpine or any other than RHEL or CentOS as they provide only rpm packages for their OSs. But I agree to wait, for now I only added notification files for alpine (same as for che sever) and for all debian / ubuntu stacks. CentOS stacks were added by RHT team so I expect that they are "correct" but yes we need some approval on that. 
And also we have 1 stack which use OracleJDK which is NOT ok. @eivantsov  Told me that dot net stack does not work with openJDK so we will investigate later.

**Comment by TylerJewell** [comment]
Yep, +1

**Comment by eivantsov** [comment]
Yes, dontnet image is using Oracle JDK since openjdk brings some packages that cause dotnet failures. We need to find out what packages are those and how to fix it.

**Comment by riuvshin** [comment]
@TylerJewell we've fixed all stacks.
there is 3 category of stacks:
- CentOS / Fedora - OpenJDK installed from repos official repos,  not additional license/notification files required (CONFIRMED)
- NO JDK installed / needed - images where we just don't have any JDK like `mysql` stack
- Rest images updated with concrete OpenJDK version + notification file with link to OpenJDK sources for particular OS added.

So I think that's it for stacks.

**Comment by TylerJewell** [comment]
Yeah!!! Ok, nice job.  Merge when you have reviewers sign off.

**Pull request merged**